src/index.js

```diff
@@ -191,12 +199,26 @@ export const Table = forwardRef(({
     useSortBy,
     usePagination,
   )
+
+  const [isOverflow, setIsOverflow] = useState(true)
+
+  const tableRef = useRef(null)
+  const tableContainerRef = useRef(null)
+
   const tableClasses = tableStyle({ 
-    headerTitle,
+    isOverflow,
     centerHeader: defaultStyles.centerHeader, 
     compactTable: defaultStyles.compactTable,
   })
-  const tableRef = useRef(null)
+
+  useEffect(() => {
+    const { current } = tableContainerRef
+    if (current && current.scrollWidth === current.clientWidth) {
+      setIsOverflow(false)
+    } else {
+      setIsOverflow(true)
+    }
```

Comment:
The flow condition can be the direct value:

```js
setIsOverflow(!(current && current.scrollWidth === current.clientWidth))
```

If you want to expand the encompassing negation, apply De Morgan's law:

```js
setIsOverflow(!current || (current.scrollWidth !== current.clientWidth))
```

Use whichever that looks more readable/logical